jax/_src/numpy/lax_numpy.py

```diff
@@ -961,9 +961,73 @@ def matrix_transpose(x: ArrayLike, /) -> Array:
   return lax.transpose(x, axes)
 
 
-@util.implements(np.rot90, lax_description=_ARRAY_VIEW_DOC)
 @partial(jit, static_argnames=('k', 'axes'))
 def rot90(m: ArrayLike, k: int = 1, axes: tuple[int, int] = (0, 1)) -> Array:
+  """Rotate an array by 90 degrees counterclockwise in the plane specified by axes.
+
+  JAX implementation of :func:`numpy.rot90`.
+
+  Args:
+    m: input array. Must have ``m.ndim >= 2``.
+    k: int, optional, default=1. Specifies the number of times the array is rotated.
+    axes: tuple of 2 integers, optional, default= (0, 1). The axes define the plane
+      in which the array is rotated. Both the axes must be different.
+
+  Returns:
+    An array containing the elements of ``m`` rotated by 90 degrees.
+
+  Note:
+    - Unlike :func:`numpy.rot90`, :func:`jax.numpy.rot90` will return a copy rather
+      than a view of the input array. However, under JIT, the compiler will
+      optimize-away such copies when possible, so this doesn't have performance
+      impacts in practice.
+    - For negative values of ``k``, the array is rotated in clockwise direction.
```

Comment:
This would fit better under the description of `k` rather than under a separate note.

Reply:
Done.